src/cider/nrepl.clj

```diff
@@ -478,12 +478,32 @@
               :returns {"fn-deps" "A list of function deps."
                         "status" "done"}}}})
 
+(def-wrapper wrap-clojuredocs cider.nrepl.middleware.clojuredocs/handle-clojuredocs
+  {:doc "Middleware to find a documents from ClojureDocs."
+   :handles {"load-clojuredocs-cache!"
+             {:doc "Load exported documents file from ClojureDocs, and store it as a cache."
+              :requires {}
+              :optional {"export-edn-url" "EDN file URL exported from ClojureDocs. Defaults to \"https://clojuredocs-edn.netlify.com/export.edn\"."}
+              :returns {"status" "\"ok\" if loading was successful"}}
+             "clean-clojuredocs-cache!"
```

Comment:
Same with this one - probably we can expose a single "clojuredocs-refresh-cache"  op instead of two.